client/components/happychat/style.scss

```diff
@@ -284,10 +284,25 @@
 	padding: 0 8px;
 	margin: 6px 0;
 
+	a {
```

Comment:
I think we prefer named class selectors instead of targeting HTML elements. I thought it's written in our [CSS guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md) but I can't find it now. :D